montagna/templates/single.html

```diff
@@ -1,8 +1,10 @@
-<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} -->
-<div class="wp-block-group"><!-- wp:template-part {"slug":"header","tagName":"header","align":"full"} /-->
+<!-- wp:group {"style":{"spacing":{"blockGap":"0","padding":{"right":"0","left":"0"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-right:0;padding-left:0"><!-- wp:template-part {"slug":"header","theme":"montagna","tagName":"header","align":"full"} /-->
```

Comment:
```suggestion
<div class="wp-block-group" style="padding-right:0;padding-left:0"><!-- wp:template-part {"slug":"header","tagName":"header","align":"full"} /-->
```

We don't need to include the `theme` attribute here as it's added automatically in the editor.